apps/web/instrumentation-client.ts

```diff
@@ -21,6 +21,18 @@ Sentry.init({
   // Setting this option to true will print useful information to the console while you're setting up Sentry.
   debug: !!process.env.SENTRY_DEBUG,
   beforeSend(event) {
+    if (
+      event.exception?.values?.some(
+        (e) =>
```

Comment:
Adding this handling to client config only because https://calcom.sentry.io/issues/6596200785/?project=4504055091625984&query=is%3Aunresolved%20issue.priority%3A%5Bhigh%2C%20medium%5D&referrer=issue-stream&sort=freq&stream_index=0 is a client side error